Automattic/wp-calypso
# Pie Chart Component

This PR adds a new PieChart component for creating pie charts.

## Testing

1. Go to https://calypso.live/devdocs/design/pie-chart?branch=add/pie-chart
or locally at http://calypso.localhost:3000/devdocs/design/pie-chart
1. Notice that the data sets in the legend are in descending order
1. Notice that the order of the sections in the legend corresponds to the order of the sections on the chart
